components/dashboard/src/components/DropDown2.tsx

```diff
@@ -207,3 +240,40 @@ export const DropDown2: FunctionComponent<DropDown2Props> = (props) => {
         </div>
     );
 };
+
+type DropDown2SelectedElementProps = {
+    iconSrc: string;
+    loading?: boolean;
+    title: ReactNode;
+    subtitle: ReactNode;
+    htmlTitle?: string;
+};
+
+export const DropDown2SelectedElement: FC<DropDown2SelectedElementProps> = ({
+    iconSrc,
+    loading = false,
+    title,
+    subtitle,
+    htmlTitle,
+}) => {
+    return (
+        <div className={classNames("flex items-center", loading && "animate-pulse")} title={htmlTitle}>
+            <div className="mx-2 my-3 flex-shrink-0">
+                <img className="w-8 filter-grayscale" src={iconSrc} alt="logo" />
+            </div>
+            <div className="flex-col ml-1 flex-grow">
+                {loading ? (
+                    <div className="flex-col space-y-2">
+                        <div className="bg-gray-300 dark:bg-gray-500 h-4 w-24" />
+                        <div className="bg-gray-300 dark:bg-gray-500 h-2 w-40" />
```

Comment:
**nitpick**: Minor corner radius change of the skeleton loaders.

```suggestion
                        <div className="bg-gray-300 dark:bg-gray-500 h-4 w-24 rounded" />
                        <div className="bg-gray-300 dark:bg-gray-500 h-2 w-40 rounded" />
```

| BEFORE | AFTER |
|-|-|
| <img width="492" alt="Frame 1637" src="https://github.com/gitpod-io/gitpod/assets/120486/bd5bcf7f-69ce-4926-b07b-0dad7d2ba820"> | <img width="492" alt="Frame 1638" src="https://github.com/gitpod-io/gitpod/assets/120486/5eadf925-8e91-4dcc-a905-97d7db283f0a"> |